contracts/Registry.vy

```diff
@@ -1,4 +1,4 @@
-# @version 0.2.11
+# @version 0.2.12
```

Comment:
We can't bump this one unless we re-release the registry

Reply:
👌